org2jekyll.el

```diff
@@ -152,31 +172,52 @@ POST-TITLE is the title.
 POST-DESCRIPTION is the description.
 POST-TAGS is the tags
 POST-CATEGORIES is the categories."
-  (format org2jekyll-jekyll-org-post-template blog-layout blog-author post-date (org2jekyll--yaml-escape post-title) post-description post-tags post-categories))
+  (format org2jekyll-jekyll-org-post-template blog-type blog-layout blog-author post-date (org2jekyll--yaml-escape post-title) post-description post-tags post-categories))
 
 (defun org2jekyll--draft-filename (draft-dir title)
   "Compute the draft's filename from the DRAFT-DIR and TITLE."
   (concat draft-dir (org2jekyll--make-slug title) org2jekyll-jekyll-post-ext))
 
+(defsubst org2jekyll--get-post-types ()
```

Comment:
What's defsubst? 

Answer: https://www.gnu.org/software/emacs/manual/html_node/elisp/Inline-Functions.html

They seem to recommand `defun` because speed is not that crucial.